optimum/neuron/utils/argument_utils.py

```diff
@@ -143,29 +143,32 @@ def store_compilation_config(
     input_names: List[str],
     output_names: List[str],
     dynamic_batch_size: bool,
-    neuron_compiler: str,
-    neuron_compiler_version: str,
+    compiler_type: str,
+    compiler_version: str,
     **kwargs,
 ):
     if isinstance(config, OrderedDict):
         update_func = config.__setitem__
     else:
         update_func = config.__setattr__
+    neuron_configs = {}
```

Comment:
You seem to call it `neuron_config` (singular) in other places.

Reply:
As the `neuron_configs` here is not the same thing as `neuron_config`(instance of `NeuronConfig`), I should probably not name them the same. I might just go with `config_args`.